weave/trace/serialize.py

```diff
@@ -168,12 +197,12 @@ def dictify(
                 else:
```

Comment:
Do you have an example? The ones I'm thinking of would be handled by the `callable` check.

Reply:
let's say we had something like:
```
class APIResponse():
   response_class: Type

res = APIResponse(response_class=MyBaseModel)
```

My understanding is that this would walk into MyBaseModel and traverse all of it's properties as well (which are types by that point). Consider https://github.com/openai/openai-python/blob/6e42e7879a8e22db9268d1ef2f3b80428534e752/src/openai/_response.py#L48 which is the base of all openai responses

Reply:
I think it wouldn't, as `BaseAPIResponse` is `callable`:
```
from openai._response import BaseAPIResponse
assert callable(BaseAPIResponse)
```